cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Insert/GovAction.hs

```diff
@@ -0,0 +1,352 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+
+module Cardano.DbSync.Era.Shelley.Insert.GovAction (
+  insertConstitution,
+  insertCostModel,
+  insertCredDrepHash,
+  insertDrep,
+  insertDrepDistr,
+  insertGovActionProposal,
+  insertParamProposal,
+  insertVotingProcedures,
+  insertVotingAnchor,
+  resolveGovActionProposal,
+  updateEnacted,
```

Comment:
Most of this stuff is Conway specific, so I wonder if it would make sense to create an `Era/Conway`

Reply:
The problem with `insertShelleyBlock` is that upstream it's used for all eras https://github.com/IntersectMBO/cardano-db-sync/blob/15656e42a09d7b9813729e8e284d622c77795714/cardano-db-sync/src/Cardano/DbSync/Default.hs#L155-L182

So technically we should move specific era inserts into the correct era directories and each could be in a single file, rather than the sub modules? eg:
```
Era/Byron/Insert.hs
Era/Conway/Shelley.hs
Era/Conway/Insert.hs
```
Then have `insertShelleyBlock` moved and renamed to a module say `Cardano.DbSync.Insert`